tests/__init__.py

```diff
@@ -73,13 +73,24 @@ def create_graph_helper(edges: List[Tuple[object, object]] = list()):
     return graph
 
 
+def create_coloring_helper_vertex(mapping: dict) -> Coloring:
+    """
+    Converts a dictionary to Coloring
+    """
+    coloring = Coloring()
+    for key in mapping:
+        for vertex in mapping[key]:
+            coloring.set(vertex=vertex, color=key)
+    return coloring
+
+
 def create_coloring_helper(vertices: List[int], map: dict):
-        coloring = Coloring()
```

Comment:
Ik snap nog niet helemaal het verschil tussen deze twee helper functies. Vooral omdat volgens mij `key` in de eerste functie `color` heet in de tweede functie en de tweede geen documentatie heeft over zn doel.